api/turing/cluster/servicebuilder/router.go

```diff
@@ -25,38 +25,39 @@ import (
-	envSentryDSN                    = "APP_SENTRY_DSN"
-	envResultLogger                 = "APP_RESULT_LOGGER"
-	envGcpProject                   = "APP_GCP_PROJECT"
-	envBQDataset                    = "APP_BQ_DATASET"
-	envBQTable                      = "APP_BQ_TABLE"
-	envBQBatchLoad                  = "APP_BQ_BATCH_LOAD"
-	envFluentdHost                  = "APP_FLUENTD_HOST"
-	envFluentdPort                  = "APP_FLUENTD_PORT"
-	envFluentdTag                   = "APP_FLUENTD_TAG"
-	envKafkaBrokers                 = "APP_KAFKA_BROKERS"
-	envKafkaTopic                   = "APP_KAFKA_TOPIC"
-	envKafkaSerializationFormat     = "APP_KAFKA_SERIALIZATION_FORMAT"
-	envKafkaMaxMessageBytes         = "APP_KAFKA_MAX_MESSAGE_BYTES"
-	envKafkaCompressionType         = "APP_KAFKA_COMPRESSION_TYPE"
-	envRouterConfigFile             = "ROUTER_CONFIG_FILE"
-	envRouterProtocol               = "ROUTER_PROTOCOL"
-	envGoogleApplicationCredentials = "GOOGLE_APPLICATION_CREDENTIALS"
-	envPluginName                   = "PLUGIN_NAME"
-	envPluginsDir                   = "PLUGINS_DIR"
+	envAppName                         = "APP_NAME"
+	envAppEnvironment                  = "APP_ENVIRONMENT"
+	envRouterTimeout                   = "ROUTER_TIMEOUT"
+	envEnricherEndpoint                = "ENRICHER_ENDPOINT"
+	envEnricherTimeout                 = "ENRICHER_TIMEOUT"
+	envEnsemblerEndpoint               = "ENSEMBLER_ENDPOINT"
+	envEnsemblerTimeout                = "ENSEMBLER_TIMEOUT"
+	envLogLevel                        = "APP_LOGLEVEL"
+	envFiberDebugLog                   = "APP_FIBER_DEBUG_LOG"
+	envCustomMetrics                   = "APP_CUSTOM_METRICS"
+	envJaegerEnabled                   = "APP_JAEGER_ENABLED"
+	envJaegerEndpoint                  = "APP_JAEGER_COLLECTOR_ENDPOINT"
+	envSentryEnabled                   = "APP_SENTRY_ENABLED"
+	envSentryDSN                       = "APP_SENTRY_DSN"
+	envResultLogger                    = "APP_RESULT_LOGGER"
+	envGcpProject                      = "APP_GCP_PROJECT"
+	envBQDataset                       = "APP_BQ_DATASET"
+	envBQTable                         = "APP_BQ_TABLE"
+	envBQBatchLoad                     = "APP_BQ_BATCH_LOAD"
+	envFluentdHost                     = "APP_FLUENTD_HOST"
+	envFluentdPort                     = "APP_FLUENTD_PORT"
+	envFluentdTag                      = "APP_FLUENTD_TAG"
+	envKafkaBrokers                    = "APP_KAFKA_BROKERS"
+	envKafkaTopic                      = "APP_KAFKA_TOPIC"
+	envKafkaSerializationFormat        = "APP_KAFKA_SERIALIZATION_FORMAT"
+	envKafkaMaxMessageBytes            = "APP_KAFKA_MAX_MESSAGE_BYTES"
+	envKafkaCompressionType            = "APP_KAFKA_COMPRESSION_TYPE"
+	envRouterConfigFile                = "ROUTER_CONFIG_FILE"
+	envRouterProtocol                  = "ROUTER_PROTOCOL"
+	envGoogleApplicationCredentials    = "GOOGLE_APPLICATION_CREDENTIALS"
+	envExpGoogleApplicationCredentials = "EXP_GOOGLE_APPLICATION_CREDENTIALS"
```

Comment:
Super nit: can this be `GOOGLE_APPLICATION_CREDENTIALS_EXPERIMENT_ENGINE` ? I think it's nicer if the name starts with `GOOGLE_APPLICATION_CREDENTIALS` and replacing `EXP` with `EXPERIMENT_ENGINE` would be clearer.